src/main/java/com/adyen/model/balanceplatform/AssociationInitiateRequest.java

```diff
@@ -92,6 +90,9 @@ public AssociationInitiateRequest ids(List<String> ids) {
   }
 
   public AssociationInitiateRequest addIdsItem(String idsItem) {
+    if (this.ids == null) {
```

Comment:
I don't think we should instantiate the list here, it hides initialization logic (& side effects), if we need it it should be on constructor/initilization level, what do you think?

Reply:
We have decided to initialise the containers (map/list/set) as `null`, as it used to be in previous versions (see iOS SDK incident). It is also a popular Java practise to initialise the container only if you use it.

I also prefer the initialisation at constructor level to be honest, but given the above we shouldn't change it.